src/chocolatey/infrastructure.app/configuration/ChocolateyConfiguration.cs

```diff
@@ -384,6 +384,8 @@ public sealed class SourcesCommandConfiguration
         public string Username { get; set; }
         public string Password { get; set; }
         public int Priority { get; set; }
+        public string Certificate { get; set; }
+        public string CertificatePassword { get; set; }

```

Comment:
Will you have both a user name / password and a certificate / password?

In cases like this maybe it makes sense that EncryptedPassword is used for both.